RELEASE_NOTES

```diff
@@ -32,6 +32,11 @@
    have correctness or performance issues.  Known correctness issues
    include dynamic process support and generalized request support.
 
+### Fortran
+
+ * The Fortran mpi module (use mpi) does not define explicit interfaces
+   for MPI routines with choice buffer arguments. Because of this, some
+   Fortran compilers require extra flags to build the Fortran bindings.
```

Comment:
Should we metion `use mpi_f08` since that is a complete and more modern interface?

Reply:
Yeah, that's a good idea.